tests/runtimes/cloud/runtime_test.py

```diff
@@ -116,3 +117,75 @@ def testRuntimeScanStop_whenScanIdIsValid_RemovesScanService(
     assert (
         mock_agent_group.call_args[0][2][0]["args"][3]["value"] == b'["url1", "url2"]'
     )
+
+
+def testCloudDescribeVuln_whenVulnHasExploitationAndPostExploitationDetails_printsExploitationAndPostExploitationDetails(
```

Comment:
Click does have an utilty to test a command and assert on ouput you can use that to make this unit way more simple.

Reply:
+1